### ISSUE TITLE
Update for Slim Framework 4

### ISSUE THREAD

**Issue by kAlvaro** [bryanjhv/slim-session#42]
Slim/4 [is out](https://www.slimframework.com/2019/08/01/slim-4.0.0-release.html) so this middleware library is no longer usable:

```
"require": {
    "php": ">=5.5",
    "slim/slim": "^3.0"
},
```

It's also a handicap to upgrade your app to Slim/4 since Composer complaints:

> bryanjhv/slim-session 3.6.1 requires slim/slim ^3.0 

(I've no idea of the amount of changes required.)

P.S. Perhaps you don't even need to have Slim as requirement any more :-?

**Comment by bryanjhv** [comment]
Hi, thanks for letting me know of Slim 4.
I was reading a little about what changed and it seems, as you say, that there aren't lots of changes so maybe I  could even make it work for both Slim 3 and 4.
I'll do my best on updating this as soon as possible.

**Comment by parkbroom** [comment]
thanks i am in need of this also for slim 4.  seems there no middleware available anywhere yet

**Comment by bryanjhv** [comment]
Thanks for the interest.
A new version for 4.x will be out in less than 48 hours.
Sorry for the delay and thanks for your patience.

**Comment by bryanjhv** [comment]
Implemented in 35936cd, please test it.
Feel free to report issues if there are present.
